addons/l10n_in/models/account_invoice.py

```diff
@@ -18,7 +18,7 @@ class AccountMove(models.Model):
             ('special_economic_zone', 'Special Economic Zone'),
             ('deemed_export', 'Deemed Export')
         ], string="GST Treatment", compute="_compute_l10n_in_gst_treatment", store=True, readonly=False, copy=True)
-    l10n_in_state_id = fields.Many2one('res.country.state', string="Location of supply")
+    l10n_in_state_id = fields.Many2one('res.country.state', string="Place of supply", compute="_compute_l10n_in_state_id", store=True)
     l10n_in_gstin = fields.Char(string="GSTIN")
```

Comment:
still needed?

Reply:
Yes, Why we need it. I explained it above.

Reply:
l10n_in_gstin is just a related to the shipping partner vat (or the regular partner one).  There is no need for a field for that normally.  All those _l10n_in_get_shipping_partner_gstin... can be cleaned out anyways.  @jva-odoo

Reply:
Hmm, okay, I think we can also remove this l10n_in_gstin.

Reply:
@jco-odoo 

> l10n_in_gstin is just a related to the shipping partner vat (or the regular partner one). There is no need for a field for that normally. All those _l10n_in_get_shipping_partner_gstin... can be cleaned out anyways. @jva-odoo

@jco-odoo I am removing this in another PR https://github.com/odoo/odoo/pull/81977